SimpleWAWS/Extensions.cs

```diff
@@ -73,8 +73,8 @@ public static bool IsBrowserRequest(this HttpContextBase context)
 
         public static bool IsFunctionsPortalRequest(this HttpContextBase context)
         {
-            return context.Request.UserAgent != null
-                && context.Request.UserAgent.StartsWith("Functions/", StringComparison.OrdinalIgnoreCase);
+            return context.Request.Headers["User-Agent2"] != null
+                && context.Request.Headers["User-Agent2"].StartsWith("Functions/", StringComparison.OrdinalIgnoreCase);
```

Comment:
use something other than `User-Agent2` something like `ms-x-source` or something like that.